PluginGenerator/Interfaces/IMavenArtifactHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="IMavenArtifactHandler.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+using System.Collections.Generic;
+
+namespace SonarQube.Plugins.Maven
+{
+    public interface IMavenArtifactHandler
+    {
+        /// <summary>
+        /// Returns a local path to the jar of the specified Maven artifact.
+        /// </summary>
+        /// <param name="coordinate">Identifier for a Maven artifact</param>
+        /// <returns>Local file paths to the jar</returns>
```

Comment:
Should be "path" singular (an earlier version of the interface returned multiple jars)